src/lib/validation/recipient.ts

```
@@ -1,37 +1,33 @@
 export async function validateAndResolveRecipient(
     recipient: string
 ): Promise<{ identifier: string; recipientType: RecipientType; resolvedAddress: string }> {
     const recipientType = getRecipientType(recipient)
-    let resolvedAddress: string | undefined
 
     switch (recipientType) {
         case 'ENS':
             // resolve the ENS name to address
-            resolvedAddress = await resolveFromEnsName(recipient)
-
+            const resolvedAddress = await resolveFromEnsName(recipient)
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)